roxctl/sensor/generate/generate.go

```diff
@@ -223,7 +210,7 @@ func Command(cliEnvironment environment.Environment) *cobra.Command {
 
 	c.PersistentFlags().BoolVar(&generateCmd.cluster.GetTolerationsConfig().Disabled, "disable-tolerations", false, "Disable tolerations for tainted nodes")
 
-	autobool.NewFlag(c.PersistentFlags(), &generateCmd.slimCollectorP, slimCollector, "Use slim collector in deployment bundle")
+	autobool.NewFlag(c.PersistentFlags(), &generateCmd.slimCollectorP, slimCollector, "[Deprecated] this argument is ignored")
```

Comment:
There might be a way to hide the flag so that it does not show up in `--help` but does not lead to errors when parsing the command line.

Reply:
I've finally been able to properly test this scenario and it works. So I think I can revert the changes here and just update the help text as you've suggested.